app/views/recipes/_recipe_details.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="flex flex-col items-center justify-center">
+
+	<h1 class="text-3xl text-gray-700 font-semibold text-center"><%=recipe.name%></h1>
+	<p class="font-semibold text-gray-600 text-center">Preparation time: <%=recipe.preparation_time%></p>
+	<div class="flex justify-end">
+		 <%=form_with url: recipe_path, method: :put do %>
+             public: <label class="switch">
+               <%= check_box_tag "",@recipe.public,@recipe.public, onchange: "this.form.submit()" %>
+               <span class="slider round"></span>
+             </label>
+             <%end%> 
+  </div>
+  <p class="font-semibold text-gray-600 text-center">Cooking time: <%=recipe.cooking_time%></p>
+  <p class="text-center font-semibold text-xl">Description: <%=recipe.description%></p>
+
+  <%= link_to "Back to recipes", recipes_path, class:"no-underline hover:underline text-purple-700 text-lg"%>
+
+  <div class="d-flex justify-content-between">
+    <button type="button" class="long-btn btn open-btn">Generate Shopping List</button>
+    <%= link_to "Add Ingredients", new_recipe_recipe_food_path(@recipe), class:"btn"%>
+  </div>
+  <table class="table table-striped table-dark ">
+    <thead>
+      <tr class="divide-x">
+        <th class="flex align-left pl-1">Food</th>
+        <th class="ml-0">Quantity</th>
+        <th class="flex pl-1">Value</th>
+        <th>Actions</th>
+      </tr>
+    </thead>
+
+    <tbody>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement [previous reviewer's suggestion](https://github.com/DaveZag/recipe-app/pull/22/files#r1111062706), kindly I also can't open `modal`, you know project need to has a `modal` to choose an inventory to create a shopping list with, as in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food_shopping_list.png?raw=true). Please implement it 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/219884512-2e7815a9-2001-490d-bf6d-c41071a0d4d0.png)

Reply:
Hi @mukhammadrizooff, thanks for the feedback. I've already implemented the required changes. May be you should `pull`🙂